get_id.js

```diff
@@ -1,104 +1,164 @@
-    console.log("Thermoplastic polymer")
-} else if (id_type_material === 2) {
-    console.log("Additives")
-} else if (id_type_material === 3) {
-    console.log("Thermoplastic elastomer")
-    if (id_type_of_filler === 1) {
-        console.log("Thermoplastic composite materials => Disperse filler")
-        parse_TKM_disperse_filler(resp, TKM_disperse_filler)
-    } else if (id_type_of_filler === 2) {
-        console.log("Thermoplastic composite materials => Fibrous filler")
+    resp = resp[0]
+    console.log(resp)
+    let id_type_material = resp.id_type_material
+    let id_type_of_filler
+    switch (id_type_material) {
+        case 1: {
+            console.log("Thermoplastic polymer")
+            parse_termoplastic_polymer(resp)
+        }
+        case 2: {
+            console.log("Additives")
+            parse_additive(resp)
+        }
+        case 3: {
+            console.log("Thermoplastic elastomer")
+            parse_termoplastic_elastomer(resp)
+        }
+        case 4:{
+            id_type_of_filler = resp.id_type_of_filler
+            switch (id_type_of_filler) {
+                case 1: {
+                    console.log("Disperse filler")
+                    parse_disperse_filler(resp)
+                }
+                case 2: {
+                    console.log("Fibrous filler")
+                    parse_fibrous_filler (resp)
+                }
+                case 3: {
+                    console.log("Other filler")
+                    parse_other_filler(resp)
+                }
+            }
+        }
+        case 5: {
+            id_type_of_filler = resp.id_type_of_filler
+            switch (id_type_of_filler) {
+                case 1: {
+                    console.log("Thermoplastic composite materials => Disperse filler")
+                    parse_TKM_disperse_filler(resp)
+                }
+                case 2: {
+                    console.log("Thermoplastic composite materials => Fibrous filler")
+                    parse_TKM_fibrous_filler(resp)
+                }
+
+        }
     }
-} else if (id_type_material === 4) {
-    console.log("Fillers")
-    id_type_of_filler = resp.id_type_of_filler
-    if (id_type_of_filler === 1) {
-        console.log("Disperse filler")
-        parse_disperse_filler(resp, filler_dispers)
-    } else if (id_type_of_filler === 2) {
-        console.log("Fibrous filler")
-        parse_fibrous_filler (resp, filler_fibrous)
-    } else if (id_type_of_filler === 3) {
-        console.log("Other filler")
-        parse_other_filler(resp, filler_other)
     }
-} else if (id_type_material === 5) {
-    console.log("Thermoplastic composite materials")
 }
 
 
-
-function fill_form(name, placeholder) {
+function put_placeholder(name, placeholder) {
     let selector = $(`input[name=${name}]:visible`)
-    selector[0].placeholder = placeholder
+    if (selector[0] !== undefined) {
+        if (selector[0].placeholder !== undefined) {

```

Comment:
It is wrong.  Input may not has a placeholder but fun must to write it 
We need to talk about this